hack/cluster-push-prep.sh

```diff
@@ -9,13 +9,17 @@
 
 set -xeuo pipefail
 
-oc -n openshift-cluster-version scale --replicas=0 deploy/cluster-version-operator
+# XXX: --type merge completely overrides any previous "overrides" array
+#      find a way to just append? json op: add isn't working at all
+#      if there's not an overrides array already, that's why we use merge
+oc patch clusterversions.config.openshift.io/version --type merge -p '{"spec":{"overrides": [{"kind": "Deployment","name": "machine-config-operator", "namespace": "openshift-machine-config-operator", "unmanaged": true}, {"kind": "ConfigMap","name": "machine-config-operator-images", "namespace": "openshift-machine-config-operator", "unmanaged": true}]}}'
```

Comment:
I believe we don't need to add the Deployment for the operator here or we're going to miss the wiring with the CVO (status report and whatnot). @cgwalters is this your understanding as well?